behat_ibexa_commerce.yaml

```diff
@@ -2,6 +2,7 @@ imports:
     - behat_ibexa_experience.yaml
     - vendor/ibexa/commerce-admin-ui/behat_suites.yml
     - vendor/ibexa/commerce-shop/behat_suites.yml
+    - vendor/ibexa/product-catalog/behat_suites.yml
```

Comment:
We can remove this line - we already import the Experience file here and the product-catalog file will be included in that import.